### DISCUSSION TITLE
Enable when you interact with Controlnet from blank (Settings Option) UI/UX

### DISCUSSION THREAD

**Discussion by SeanReynolds** [Mikubill/sd-webui-controlnet#1703] [Ideas]
Just something that seems to be a thing everyone in communities do at least once upon loading up. Workflow wise, you load up, put your settings in then choose what you want controlnet to do, set your run and batches but you forgot to tick enabled, so it ignores CN settings and you find out a few minutes later.

Inane QOL option but if I could choose that the first time you interacted with controlnet by adding an image to it from blank it automatically assumed I was activating it, it would be helpful!

**Comment by huchenlei** [comment]
The feature seems to be a good QoL improvement and not hard to implement. Do note that if you do not pass any image to ControlNet, the A1111 img2img input will be used as fallback though.

**Reply by huchenlei** [reply]
Implemented in #1704.
